### PR TITLE
SEAL-6309 Add handling for list methods in DmnJsonConverter processRule

### PR DESCRIPTION
**Pull request by AminaZoheir**

### CODE REVIEW

File: modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java [hunk 1]
```diff
@@ -363,25 +425,67 @@ protected void processRules(JsonNode modelNode, Map<String, InputClause> ruleInp
...
+                                stringBuilder.append(expressionValue);
+                                stringBuilder.append("')");
+                            }
+                            String expression = stringBuilder.toString();
+                            stringBuilder = new StringBuilder(name);
+                            if (listOperatorValue.toLowerCase().equals("append")) {
+                                stringBuilder.append(".append(");
+                                stringBuilder.append(expression);
+                                stringBuilder.append(")");
+                            } else if (listOperatorValue.toLowerCase().equals("remove")) {
```
Comment (ghonia) [inline modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java]: removeObject

File: modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java [hunk 2]
```diff
@@ -363,25 +425,67 @@ protected void processRules(JsonNode modelNode, Map<String, InputClause> ruleInp
...
+                                stringBuilder.append(".remove(");
+                                stringBuilder.append(expression);
+                                stringBuilder.append(")");
+                            } else if (listOperatorValue.toLowerCase().startsWith("clear")) {
```
Comment (ghonia) [inline modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java]: clearMetadata?

File: modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverterUtil.java [hunk 3]
```diff
@@ -190,8 +302,13 @@ public static String determineExpressionType(String expressionValue) {
                     if ("true".equalsIgnoreCase(expressionValue) || "false".equalsIgnoreCase(expressionType)) {
                         expressionType = "boolean";
                     }
+                    if(expressionValue.startsWith("\"")) {
+                        expressionType = "string";
+                    }
                 }
```
Comment (ghonia) [inline modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverterUtil.java]: Mafish date?

File: modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java (hunk 1 above)
Comment (AminaZoheir) [reply modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java]: This is for the check it checks on remove and sets the method to removeObject.
Not sure if this comment was before or after the change

File: modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java (hunk 2 above)
Comment (AminaZoheir) [reply modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverter.java]: Same as the one above

File: modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverterUtil.java (hunk 3 above)
Comment (AminaZoheir) [reply modules/flowable-dmn-json-converter/src/main/java/org/flowable/editor/dmn/converter/DmnJsonConverterUtil.java]: The date was already there so it just isn't visible in the diff

**Pull request merged**